qiskit/transpiler/preset_passmanagers/level2.py

```diff
@@ -100,8 +101,27 @@ def level_2_pass_manager(pass_manager_config: PassManagerConfig) -> PassManager:
     def _choose_layout_condition(property_set):
         return not property_set['layout']
 
+    _choose_layout_0 = [] if pass_manager_config.layout_method \
+        else [TrivialLayout(coupling_map),
+              Layout2qDistance(coupling_map,
+                               property_name='trivial_layout_score')]
     _choose_layout_1 = [] if pass_manager_config.layout_method \
         else CSPLayout(coupling_map, call_limit=1000, time_limit=10, seed=seed_transpiler)
+
+    def _not_perfect_yet(property_set):
```

Comment:
`trivial_not_perfect` might be easier to read.

Also, it would be helpful to comment or document how these are expected to behave in the cases where the user provides an initial layout or layout method. It's not necessarily apparent on first read through to figure out why e.g. `_not_perfect_yet` doesn't need to check `property_set['layout']`.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/5702/commits/9bb854902990bf0bb05c61bc4e39e5962f9b189e